server/main.lua

```diff
@@ -345,9 +310,69 @@ local function spawnVehicle(src, data)
     return netId
 end
 
-lib.callback.register('qbx_vehicleshop:server:spawnVehicle', spawnVehicle)
+---@param data {vehicle: string}
+RegisterNetEvent('qbx_vehicleshop:server:testDrive', function(data)
+    local src = source
+
+    if Player(src).state.isInTestDrive then
+        return exports.qbx_core:Notify(src, locale('error.alreadytestdriving'), 'error')
```

Comment:
```suggestion
        return exports.qbx_core:Notify(src, locale('error.testdrive_alreadyin'), 'error')
```
This is an invalid locale